source/npm/qsharp/src/compiler/compiler.ts

```diff
@@ -188,6 +172,7 @@ export class Compiler implements ICompiler {
     // All results are communicated as events, but if there is a compiler error (e.g. an invalid
     // entry expression or similar), it may throw on run. The caller should expect this promise
     // may reject without all shots running or events firing.
+    log.info(`Running run with profile ${program.profile}`);
```

Comment:
Was this just for debugging? I worry they'll clutter up the Output UI (if you still need it, consider putting it at the `debug` level)